### ISSUE TITLE
Compendium are still in english with activated modules.

### ISSUE THREAD

**Issue by vt-tom** [mhilbrunner/foundryvtt-dnd5e-lang-de#7]
Versions:
Foundry v11.315
DnD 2.4.0
DnD German Translation: 2.4.1
Babele 2.5.2
libwrapper 1.12.13.0

Problem: with activated modules, the compendiums are not translated, they are still english.
So we cant translate any characters. No errors in the console.
![grafik](https://github.com/mhilbrunner/foundryvtt-dnd5e-lang-de/assets/80099175/71fd65c2-5e60-429d-bd48-58e3c3756634)

**Comment by mhilbrunner** [comment]
Thanks for reporting :) I'll look into this as soon as I have some time. In the mean time, downgrading DND 5E may be the best option.

If anyone has time to look into the issue, all hints on what broke with the update are welcome, especially if there are no console errors.

**Comment by Freezor** [comment]
Also with that update I'm not able to open any of the existing races, as they changed the item type. But so far I wasn't able to find an issue here on why this happens.

**Comment by Freezor** [comment]
I can see the following issue:

```javascript
d5e-de.js:546 Uncaught (in promise) TypeError: undefined. m.replace is not a function
[Detected 2 packages: FoundryVTT-dnd5e-DE, babele]
    at FieldMapping.convertSource [as converter] (dnd5e-de.js:546:46)
    at FieldMapping.translate (babele.js:1006:26)
    at FieldMapping.map (babele.js:989:28)
    at babele.js:1072:62
    at Array.reduce (<anonymous>)
    at CompendiumMapping.map (babele.js:1072:28)
    at TranslatedCompendium.translate (babele.js:1235:43)
    at Babele.translate (babele.js:763:19)
    at babele.js:78:54
    at Array.map (<anonymous>)
    at 🎁ClientDatabaseBackend.prototype._getDocuments#babele (babele.js:77:27)
    at async CompendiumCollection.getDocuments (foundry.js:16280:23)
    at async CompendiumCollection.getDocument (foundry.js:16252:23)
    at async Compendium._onClickEntryName (foundry.js:70218:57)
```

looks like this is a babele problem maybe.

I already opened an issue there: https://gitlab.com/riccisi/foundryvtt-babele/-/issues/88

**Comment by durandom** [comment]
> Thanks for reporting :) I'll look into this as soon as I have some time. In the mean time, downgrading DND 5E may be the best option.
> 
> If anyone has time to look into the issue, all hints on what broke with the update are welcome, especially if there are no console errors.

Do you mean downgrading the dnd5e core module or the translation module would fix this?

**Comment by mhilbrunner** [comment]
As long as the translation module hasn't updated, playing with it requires downgrading the 5E core module to a compatible version (which, to be clear, should mean reverting to a backup - it is not a good idea to switch to a previous system version for an existing world that was already updated to a later version).

Alternatively, you can play with the current system version but without the translation module until the latter is updated, which I should hopefully find some time for soon-ish.

**Comment by mhilbrunner** [comment]
I just released version [2.5.0](https://github.com/mhilbrunner/foundryvtt-dnd5e-lang-de/releases/tag/2.5.0) which should fix this issue. Please update the package and let me know if you run into any more issues. Thanks for helping out!

**Comment by Freezor** [comment]
Hi @mhilbrunner thanks for the fix. At least for me that did the magic
